lib/Test/Nginx.pm

```diff
@@ -39,6 +39,12 @@ L<Test::Nginx::Socket::Lua::Stream>
 
 =item *
 
+L<Test::Nginx::Socket::Lua::Dgram>
+
+=back
+
+=item *
```

Comment:
I think you should put `=item` between `=over` and `=back`. This is why the CI testing is failing in this PR.